app/Controllers/feedController.php

```diff
@@ -417,6 +419,12 @@ public static function actualizeFeed(int $feed_id, string $feed_url, bool $force
 					if ($simplePie === null) {
 						throw new FreshRSS_Feed_Exception('XML+XPath parsing failed for [' . $feed->url(false) . ']');
 					}
+				} elseif ($feed->kind() === FreshRSS_Feed::KIND_RSS_MAYBE) {
+					$simplePie = $feed->load(false, $feedIsNew);
+					// @phpstan-ignore-next-line
```

Comment:
question:  why does phpstan have to ignore this line?

Reply:
After `$feed->load()` then `$feed->kind()` might have changed, but PHPStan does not seem to recognise that and gives an error:
> Strict comparison using !== between 3 and 3 will always evaluate to false.